core/sequencer/sequencer.go

```diff
@@ -350,9 +352,28 @@ func (s *Sequencer) CreateEpoch(ctx context.Context, forceNewEpoch bool) error {
 	mapUpdateHist.Observe(mapSetEnd.Sub(mapSetStart).Seconds())
 	createEpochHist.Observe(time.Since(start).Seconds())
 	glog.Infof("CreatedEpoch: rev: %v, root: %x", revision, setResp.GetMapRoot().GetRootHash())
+
+	// store smr in BFTKV
+	writeToBFTKV(string(s.logID) + "|" + string(revision), string(setResp.GetMapRoot().GetRootHash()))
```

Comment:
Monitors seem to be storing map roots. Wouldn't it be easier if BFTKV stores the map roots and monitors compare their version of the map roots with what is stored in BFTKV?